content/docs/0.8.x/integrations/sli_provider/index.md

```diff
@@ -18,69 +18,133 @@ indicators:
+3. **Send a started event:** If the data source is supported, send a start event of type: `sh.keptn.event.get-sli.started`. This CloudEvent informs Keptn that your service takes care of fetching the SLIs. 
 
-**Distributor:** To subscribe your SLI-provider to the `sh.keptn.internal.event.get-sli` event, a distributor is required. A distributor comes with a deployment manifest as shown by the example below:
 
-```yaml
-## dynatrace-sli-provider: sh.keptn.internal.event.get-sli
-apiVersion: apps/v1
-kind: Deployment
-metadata:
-  name: dynatrace-sli-provider-distributor
-  namespace: keptn
-spec:
-  selector:
-    matchLabels:
-      run: distributor
-  replicas: 1
-  template:
-    metadata:
-      labels:
-        run: distributor
-    spec:
-      containers:
-      - name: distributor
-        image: keptn/distributor:0.7.3
-        ports:
-        - containerPort: 8080
-        resources:
-          requests:
-            memory: "32Mi"
-            cpu: "50m"
-          limits:
-            memory: "128Mi"
-            cpu: "500m"
-        env:
-        - name: PUBSUB_URL
-          value: 'nats://keptn-nats-cluster'
-        - name: PUBSUB_TOPIC
-          value: 'sh.keptn.internal.event.get-sli'
-        - name: PUBSUB_RECIPIENT
-          value: 'dynatrace-sli-provider'
-```
+4. Get the SLI configuration from Keptn's configuration-service. This SLI configuration is identified by the `resourceURI`, which follows the pattern: `[tool-name]/sli.yaml` (e.g., `dynatrace/sli.yaml`). 
+  * Service URL: http://configuration-service.keptn.svc.cluster.local:8080
+  * Endpoint: `v1/project/{projectName}/stage/{stageName}/service/{serviceName}/resource/{resourceURI}`
 
-To configure this distributor for your *SLI-provider*, the environment variables `PUBSUB_RECIPIENT` has to refer to the service name of the SLI-provider in the Kubernetes service manifest. Besides, make sure the environment variable `PUBSUB_TOPIC` has the value `sh.keptn.internal.event.get-sli`.
\ No newline at end of file
+5. Process the SLI configuration and use the defined queries to retrieve the values of each indicator. 
+
+6. **Send a finished event:** Send a finished event of type: `sh.keptn.event.get-sli.finished`. The data block of the event payload must provide at least a value for `status` and `result` in `data.get-sli`:
+
+  * `status`: [succeeded, errored, unknown] - The status of the task execution 
+  * `result`: [pass, failed] - The result of a successful task execution
+  * `indicatorValues`: List of indicators and their measured values
+
+```json
+{
+  "type": "sh.keptn.event.get-sli.finished",
+  "specversion": "1.0",
+  "source": "https://github.com/keptn/keptn/dynatrace-sli-service",
+  "id": "ggb878d3-03c0-4e8f-bc3f-454bc1b3d888",
+  "time": "2019-06-07T07:02:15.64489Z",
+  "contenttype": "application/json",
+  "shkeptncontext": "08735340-6f9e-4b32-97ff-3b6c292bc509",
+  "triggeredid": "2b878d3-03c0-4e8f-bc3f-454bc1b3d79d",
```

Comment:
let's add a short description on how to set the triggeredid

Reply:
added a description